### PR TITLE
Feat/local cache

### PR DESCRIPTION
**Pull request by CocoLi7777**

### CODE REVIEW

File: client/script.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
   async function init() {
     var rawData = await callRemoteApi(endpoint);
-    console.log(rawData.locations);
+    //console.log(rawData.locations);
```
Comment (tiagralee) [inline client/script.js:23]: Remove this line

File: client/script.js [hunk 2]
```diff
@@ -98,7 +98,7 @@
       }
     );
     lastTenDaysNewCaseValues = lastTenDaysNewCaseValues.slice(1, 11);
-    console.log(lastTenDaysNewCaseValues);
+    //console.log(lastTenDaysNewCaseValues);
```
Comment (tiagralee) [inline client/script.js:101]: remove this line

File: src/fs.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+
+// var checkTime = fileName => {
+//   var stats = fs.statSync(fileName);
+//   return stats.mtime;
+// };
```
Comment (tiagralee) [inline src/fs.js:6]: remove unused comments

File: src/routes/router.js [hunk 4]
```diff
@@ -1,27 +1,46 @@
...
+      res.send(JSON.stringify(response.data));
+      const lastModifyDate = await lastModifiedDate('localCache.json');
+      const lastModifyMonth = lastModifyDate.getMonth() + 1;
+      const lastModifyDay = lastModifyDate.getDate();
+      const currentDate = new Date().getDate();
+      const currentMonth = new Date().getMonth() + 1;
+      if (
+        lastModifyDate == 'undefined' ||
+        lastModifyMonth != currentMonth ||
+        lastModifyDay != currentDate
```
Comment (tiagralee) [inline src/routes/router.js:27]: no need to compare the datetime like this, just formate datetime to string and compare, like
lastModifyDate.toLocaleDateString("en-US") != currentDate.toLocaleDateString("en-US")

File: src/routes/router.js [hunk 5]
```diff
@@ -1,27 +1,46 @@
...
+      ) {
+        writeToLocalCache(response.data, 'localCache.json');
+      }
+    })
+    .catch(error => {
+      console.log(error);
```
Comment (tiagralee) [inline src/routes/router.js:33]: remove this line

File: src/routes/router.js (hunk 4 above)
Comment (CocoLi7777) [reply src/routes/router.js:27]: That's good shot! Thanks Al~

**Pull request merged**